csrc/transformer/inference/csrc/layer_norm.cu

```diff
@@ -200,7 +200,11 @@ Template arg:
     StoreResidual: controls whether the residual calculation is stored
         or not. When set to false, the input `res_output` is unused.
 */
-template <typename T, int UNROLL, bool StoreResidual>
+template <typename T, int UNROLL,
+          int internal_unroll,
+          int threads_per_group,
+          int max_threads,
+          bool StoreResidual>
```

Comment:
Let's refactor `StoreResidual` to `preLN` style argument to make it more clear why the store is happening (I believe you already made this change internally for quant).

Reply:
changed a few names; quant style may not be possible here since we have only one output when it is post residual layernorm. please let me know if this change looks good

Reply:
This looks good, I just meant more broadly that referring to pre-LN makes it a little more clear why we want to store the residual so what you did works great.